apps/frontend/package.json

```
@@ -18,6 +18,7 @@
     "i18next-browser-languagedetector": "7.0.0",
     "lodash.get": "4.4.2",
     "lodash.set": "4.3.2",
+    "mathjs": "^11.3.3",
```

```suggestion
    "mathjs": "11.3.3",
```
And I prefer us using ethers BigNumber instead of mathjs library from now on. It does not have decimal support, but we can still make it work

---

The main problem is with decimal support. I'm not sure how is best to handle that?

---

Workaround would be to use `parseEthers` / `parseUnits` or `formatEthers` / `formatUnits` utils from ethers.
Ofc, if you want to render number with decimals, most likely you will need to wrap your stuff with Number:
`Number(formatEthers(100000)).toFixed(8);`

---

Agreed to resolve this in [SOV-1086](https://sovryn.atlassian.net/browse/SOV-1086)